DominantColor/Shared/KMeans.swift

```diff
@@ -106,6 +106,6 @@ private extension Array {
             indices.append(random)
         }
 
-        return indices.map { self[$0] }
+        return indices.map { self[$0 - 1] }
```

Comment:
Could you comment on this change? This seems like a behavioral change rather than a compiler error fix.

Reply:
I noticed that `randomValues(_:seed:)` crashed with an index out of bounds and 'fixed' the issue by bringing it back into bounds. Upon further investigation it seems the method only crashes for some images.

The crashes are consistent, but honestly I'm not familiar enough with the code that I can fix it straight away. Thus I've removed the change and left it as is.